IFC 4.3/Align_2/readme.md

```diff
@@ -0,0 +1,30 @@
+## *Align_2*
+
+### Intent
+
+*This scenario shows an alignment with zero length circular curve between clothoids, zero length line segment between cuves, and a transition curve between two circular curves. Those zero length elements are not present in ifc file, they are just a concept.*
```

Comment:
In order to be even more clear about the content, what about adding a table here, that lists the elements in ascending order? Sth like:

```suggestion
*This scenario shows an alignment with zero length circular curve between clothoids, zero length line segment between cuves, and a transition curve between two circular curves. Those zero length elements are not present in ifc file, they are just a concept.*

| Element | Length | Entity in file |
|----|----|----|
| straight | 450m | `#35` |
```

The entries you talk about having zero length, just put -not in file- to be more clear about what you mean by that.
(I've made the first entry in the table up, I did not look in the file, please update with real data.)

Reply:
Table Done